packages/engine/Source/Scene/Model/ModelSceneGraph.js

```diff
@@ -433,8 +539,8 @@ function traverseAndCreateSceneGraph(sceneGraph, node, transformToRoot) {
   // Create and store the public version of the runtime node.
   const name = node.name;
   if (defined(name)) {
-    const model = sceneGraph._model;
     const publicNode = new ModelNode(model, runtimeNode);
+    // TODO: Why does `_nodesByName` live on the model and not here?
```

Comment:
no reason this cannot live on ModelSceneGraph instead of Model. 

The Model.getNode uses the property and can remain as is.